buildlib/pr/main.yml

```diff
@@ -40,27 +40,22 @@ resources:
       options: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools
     - container: centos7_cuda11
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos7-mofed5-cuda11:1
-      options: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools --gpus all
+      options: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools --gpus all --device=/dev/infiniband/uverbs1
     - container: centos8_cuda11
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos8-mofed5-cuda11:1
-      options: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools --gpus all
+      options: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools --gpus all --device=/dev/infiniband/uverbs1
     - container: ubuntu16_cuda11
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/ubuntu16.04-mofed5-cuda11:1
-      options: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools --gpus all
+      options: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools --gpus all --device=/dev/infiniband/uverbs1
     - container: ubuntu18_cuda11
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/ubuntu18.04-mofed5-cuda11:1
-      options: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools --gpus all
+      options: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools --gpus all --device=/dev/infiniband/uverbs1
     - container: ubuntu20_cuda11
       image: rdmz-harbor.rdmz.labs.mlnx/ucx/ubuntu20.04-mofed5-cuda11:1
-      options: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools --gpus all
+      options: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools --gpus all --device=/dev/infiniband/uverbs1
```

Comment:
1. why only uverbs1 and not also uverbs0?
2. add rdmacm
3. Is it possible to do something like `--device/dev/infiniband/*` ?

Reply:
1. added
2. added
3. docker: Error response from daemon: error gathering device information while adding custom device "/dev/infiniband/*": no such file or directory.